src/App.js

```diff
@@ -29,59 +29,43 @@ class App extends React.Component {
   }
 
     loadPeopleFromServer = (callback) => {
```

Comment:
I noticed these 4 functions have identical bodies, they only vary in the API used and (in componentDidMount) what variable their data is assigned to. Should I refactor to a single function that takes the API and the destination variable as parameters, performs the fetch, and assigns the data?